lib/account/pages/login_page.dart

```diff
@@ -129,27 +131,11 @@ class _LoginPageState extends State<LoginPage> {
                 isLoading = false;
               });
 
-              SnackBar snackBar = SnackBar(
-                content: Text('Login failed, please try again (${state.errorMessage ?? 'Unknown'})'),
-                behavior: SnackBarBehavior.floating,
-              );
-
-              WidgetsBinding.instance.addPostFrameCallback((timeStamp) {
-                ScaffoldMessenger.of(context).clearSnackBars();
-                ScaffoldMessenger.of(context).showSnackBar(snackBar);
-              });
+              showSnackbar(context, AppLocalizations.of(context)!.loginFailed(state.errorMessage ?? 'Unknown'));
```

Comment:
Maybe we can use `missingErrorMessage` localization message for the default here